packages/docs/package.json

```diff
@@ -13,8 +13,8 @@
 		"build:search": "node generateSearchIndex.mjs",
 		"build:netlify-dp": "vite build --emptyOutDir --outDir dist && typedoc --tsconfig ../core/tsconfig.json ../core/src/index.ts --out dist/api",
 		"files:pages": "touch ../../pages/.nojekyll && echo 'charts.carbondesignsystem.com' > ../../pages/CNAME",
-		"build": "yarn build:search && yarn build:docs && yarn files:pages && yarn build:api:docs",
-		"dev": "vite --port 3000",
+		"build": "yarn build:docs && yarn files:pages && yarn build:api:docs",
+		"dev": "yarn build:search && vite --port 3000",
```

Comment:
Removed building of search index from docs package build. Ensure index is rebuilt before previewing site.